src/content/docs/data-apis/ingest-apis/metric-api/cardinality-management.mdx

```diff
@@ -149,6 +149,25 @@ To delete a pruning rule:
 1. Click **View All Pruning Rules**.
 2. Scroll to the relevant pruning rule, and then click **Delete**. 
 
+## Per-metric limit adjustment[#premetric-limit-adjustment]
+
+When all attributes of a specific metric are crucial and you approach your cardinality limit, you can adjust the per-metric limit to meet your needs. Typically, metrics have a default maximum cardinality of 100,000, but you can increase this to as much as 1 million per metric.
+
+<Callout variant="tip">
+  Your account's overall cardinality limit still applies, so setting all your metric limits to 1 million without consideration could quickly exhaust your account's cardinality capacity.
+</Callout>
+
+### Edit per-metric limit [#edit-premetric-limit]
+
+1. Go to the specific metric's page and select <DNT>**Edit Per Metric Limit**</DNT>.
+    Or
+   Select <DNT>**Edit Per Metric Limit**</DNT> from the kebab menu of the desired metric in the **All Metrics** table of the **Account** Page.
+2. Enter the new desired limit for the metric in the form of a whole number.
+3. Update the value.
```

Comment:
When they enter the new limit in line 165, haven't they already made the update? Why this step?